packages/wallet-core/src/store/actions/sendNFTTransaction.ts

```diff
@@ -13,8 +13,11 @@ export const sendNFTTransaction = async (
   const account: Account = getters.accountItem(accountId)!;
   const asset = getChain(network, account.chain).nativeAsset[0].code;
   const client = getters.client({ network, walletId, chainId: account.chain });
+
   const tx = await client.nft.transfer(nft.asset_contract!.address!, receiver, [nft.token_id!], values);
 
+  commit.REMOVE_NFT_FROM_STATE({ network, walletId, accountId, nft });
```

Comment:
This method is innaccurate because it does not portray the real status of NFTs in your wallet. 
Let's assume you make the `send` call, and the tx does not confirm or gets dropped. Your NFT will still be gone from state. 

The accurate way to do this is to refresh the list of NFTs after a `sendNFT` tx has been confirmed
This should be in `sendNFT` `performNextAction`